### ISSUE TITLE
cxxbridge error: Failed to create symlink

### ISSUE THREAD

**Issue by jwinarske** [dtolnay/cxx#938]
Seeing the following intermittent build failure in our pipeline builds.  Ubuntu 20 based container.  It occasionally fails.

```
cxxbridge error: Failed to create symlink `/builds/-F39GCyv/2/.../target/cxxbridge/rust/cxx.h` pointing to `/builds/-F39GCyv/2/.../.cargo/registry/src/github.com-1ecc6299db9ec823/cxx-1.0.49/include/cxx.h`

 Caused by:

File exists (os error 17)
```

Could be file is not synchronized to disk, and the create symlink call is reading through to disk?

**Comment by dtolnay** [comment]
Sounds like a race condition! Sorry about the flakiness. I've published a fix in cxx-build 1.0.55.

**Comment by jwinarske** [comment]
Thanks!  We'll see how it goes and re-open if it pops up again.
